lib/chef/azure/chefhandlers/exception_handler.rb

```diff
@@ -33,7 +33,15 @@ def load_run_list
       first_boot = File.read("#{bootstrap_directory}/first-boot.json")
       first_boot = JSON.parse(first_boot)
       run_list = first_boot["run_list"]
-      node.run_list.reset!(run_list)
+      nodeObj = Chef::Node.load(node.name)

```

Comment:
We should add a comment indicating that we can't use validatorless bootstraps to solve this problem and we're implementing our own version of "always save the runlist" since we have to use a validator.